MREUnityRuntime/MREUnityRuntimeLib/UnityConstants.cs

```diff
@@ -4,6 +4,6 @@ namespace MixedRealityExtension
 {
 	internal static class UnityConstants
```

Comment:
>internal static class UnityConstants [](start = 1, length = 36)

The main purpose for this file was for using it for layers.  Since we are taking a different approach to that, this file should be removed.